docs/rfcs/0001-expose-object.md

```diff
@@ -0,0 +1,43 @@
+# Summary
+
+Provide the build facility to expose objects of nebula for outside project.
+
+# Motivation
+
+Some outside project will use some components of nebula directly, such as cpp client will reference the `common` and `interface` code. And also useful to `UDF` and `extension` system etc. developing in future.
+
+# Usage explanation
+
+For the module user, just try these steps:
+
+1. Include the nebula project, build the objects you need, such as `cmake --build --target base_obj` will build the object contains all code in `base` source folder.
```

Comment:
1. Include the nebula project, build the objects you need, such as `cmake --build --target base_obj` which will build the object containing all code in the `base` source folder.